data.json

```
@@ -593,6 +593,15 @@
     "dateAdded": "2019-10-04T16:13:00.000Z",
     "tags": ["hacktoberfest", "stickers", "accessories", "expired"]
   },
+  {
+    "name": "Vonage",
+    "difficulty": "medium",
+    "description": "First, sign up on the Hacktoberfest site. To qualify for the official limited edition Hacktoberfest shirt, you must register and make four pull requests between October 1-31. Pull requests can be to any public repo on GitHub, not just the ones with issues labeled Hacktoberfest. If a maintainer reports your pull request as spam or spammy behavior, or not in line with the project’s code of conduct, you will be ineligible to participate.",
+    "reference": "https://developer.nexmo.com/hacktoberfest",
+    "image": "https://res.cloudinary.com/dsg3ucw55/image/upload/v1599671200/Logo_nstaby.svg",
+    "dateAdded": "2020-10-01T10:15:00.000Z",
+    "tags": ["hacktoberfest", "stickers", "clothing"]
```

```suggestion
    "tags": ["hacktoberfest", "stickers", "clothing", "credit/coupon/gift-card"]
```
@vikaspotluri123 wdyt?

---

The tags are updated. I removed the stickers and re-added `credit/coupon/gift-card`.